database/src/pool/postgres.rs

```diff
@@ -1592,6 +1592,13 @@ where
             )
             .await
             .unwrap();
+        self.conn()
+            .execute(
+                "delete from benchmark_request where tag = $1",
```

Comment:
Nit;
```suggestion
                "DELETE FROM benchmark_request WHERE tag = $1",
```

(although  this is amongst some older SQL without keywords in caps to up to you)

Reply:
Changed the casing in the whole function.